.github/workflows/lima.yaml

```diff
@@ -53,8 +43,7 @@ jobs:
           make ARCH=arm64
           sudo make PREFIX=/opt/socket_vmnet.arm64 install.bin install.doc
           if file /opt/socket_vmnet.arm64/bin/* | grep -q x86_64 ; then false ; fi
-      - name: Update brew (macos-15-large)
-        if: matrix.platform == 'macos-15-large'
+      - name: Update brew
```

Comment:
I would keep this as well, it documents why the update is needed. If we add another platform later the unneeded step will be skippped.

Reply:
fixed